ledger/sandbox/src/main/scala/com/digitalasset/platform/configuration/MetricsReporter.scala

```diff
@@ -52,8 +54,25 @@ object MetricsReporter {
       Graphite(new InetSocketAddress(defaultHost, port))
   }
 
-  implicit val metricsReporterRead: Read[MetricsReporter] = Read.reads {
-    _.split(":", 2).toSeq match {
+  implicit val metricsReporterRead: Read[MetricsReporter] = Read.reads { value =>
+    if (value == "console") Console
+    else if (!value.contains("://")) {
+      readLegacy(value)
```

Comment:
This is not documented, the options are hidden, and I don't know of anyone using it. I don't think we need to support legacy users.

Reply:
I was mostly trying to not break any existing deployments.

Reply:
Well, I think you actually knew at least one user. :wink: https://github.com/DACH-NY/spider-on-x/pull/34